.gitignore

```diff
@@ -73,3 +73,9 @@ Examples/Tutorials/**/*.exe
 #venv
 .venv/**
 venv/**
+
+#CMake
+compile_commands.json
```

Comment:
This is more related with VS Code instead of CMake

Reply:
This the list of all compile commands CMake will execute.

Reply:
It's something about CMake, Clangd, VS Code together. CMake Generates `compile_commands.json` by "CMAKE_EXPORT_COMPILE_COMMANDS" [https://cmake.org/cmake/help/latest/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html](https://cmake.org/cmake/help/latest/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html) Clangd consumes `compile_commands.json` by set `--compile-commands-dir=` [https://clangd.llvm.org/design/compile-commands](https://clangd.llvm.org/design/compile-commands). And VS Code provides plugins such as `clangd` and `CMake Tools` to use `compile_commands.json` and analyze the whole project.

Reply:
Thanks for info